dv/uvm/icache/doc/ibex_icache_dv_plan.md

```diff
@@ -1,99 +1,142 @@
 ---
-title: "IBEX_ICACHE DV Plan"
+title: "Ibex ICache DV Plan"
 ---
 
 ## Goals
 * **DV**
-  * Verify all IBEX_ICACHE IP features by running dynamic simulations with a SV/UVM based testbench
-  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules
+  * Verify all Ibex ICache IP features by running dynamic simulations with a SV/UVM based testbench
+  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage.
 * **FPV**
-
-## Current status
-<!-- TODO for now -->
-* [Design & verification stage]({{< relref "doc/project/hw_dashboard" >}})
-  * [HW development stages]({{< relref "doc/project/hw_stages" >}})
```

Comment:
similarly, this should link to [opentitan/doc/project/hw_stages.md](https://github.com/lowRISC/opentitan/blob/master/doc/project/hw_stages.md) to conform with all other DV plan documents

Reply:
As a fully rendered example that you can look at, see the [AES dv plan](https://docs.opentitan.org/hw/ip/aes/doc/dv_plan/).

Reply:
As above. I'm not particularly keen on adding static links to OpenTitan documentation from Ibex code - the dependency arrows are supposed to go the other way around.